test/fivetran-destination/data/02-verify.json

```diff
@@ -0,0 +1,2 @@
+{}
+// FAIL: user lacks "CREATE" privilege on database ("test")
```

Comment:
Is this supposed to be part of the PR?

Reply:
This was the result of running the `fivetran-destination-tester` tests locally, I'll remove it from this PR though!